lib/src/review_service/review_condition_interface.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:review_service/src/review_service/review_settings.dart';
+
+/// A condition used to determine if a review should be requested based on [ReviewSettings].
+abstract interface class IReviewCondition<TReviewSettings extends ReviewSettings> {
```

Comment:
Shouldn't we remove the capital I at the start of interface names? It seems like it isn't the standard in Dart?

Reply:
We can, like we did on the Flutter Application Template.